localization/twist_estimator/pose2twist/src/pose2twist_core.cpp

```diff
@@ -70,7 +70,8 @@ geometry_msgs::msg::TwistStamped calcTwist(
   geometry_msgs::msg::PoseStamped::SharedPtr pose_a,
   geometry_msgs::msg::PoseStamped::SharedPtr pose_b)
 {
-  const double dt = (pose_b->header.stamp.sec - pose_a->header.stamp.sec);
+  const double dt =
+        (rclcpp::Time(pose_b->header.stamp) - rclcpp::Time(pose_a->header.stamp)).seconds();
 
```

Comment:
Is the format correct?

Reply:
Thanks! I push the commit with the format.